src/js/base/module/Editor.js

```diff
@@ -498,34 +511,6 @@ export default class Editor {
     return this.getLastRange();
   }
 
-  /**
```

Comment:
I don't think these deletion of lines were your intention.
You might have to rebase or update your local repository.

Reply:
Yeah, I think that may have unintentional, I'll remove the branch and redo it, I'm confused as to why these two edits you've mentioned have occured.